GarNet/garnet.py

```diff
@@ -135,38 +96,6 @@ def parse_known_genes_file(known_genes_file, kgXref_file=None, organism="hg19"):
 	return known_genes_dataframe
 
 
-def parse_motifs_file(motifs_file, organism="hg19"):
```

Comment:
Why did you choose to remove this?

Reply:
It's not needed if formatted correctly.

Reply:
The motifs file we get originally is not a BED file is the issue -- we need to parse it at GarNet file construction time, whether the GarNet file is a pickle of interval trees or a BED file, right?

Reply:
It was just a matter of rearranging columns, which can be done once. You cleaned and normalized the MotifMap file anyways, so rearranging columns should not require too many extra steps.

Reply:
Okay, so you're suggesting removing this and just preserving it in the notebooks? Is that right? Because there's definitely still relevant code in this function, I think you'll agree with me, to generate a proper GarNet file from Motifmap and the reference genome...

Reply:
ah right. added it back